host_core/lib/host_core/vhost/config_plan.ex

```diff
@@ -63,6 +63,8 @@ defmodule HostCore.Vhost.ConfigPlan do
           {:structured_log_level, "WASMCLOUD_STRUCTURED_LOG_LEVEL",
            required: false, map: &string_to_loglevel/1},
           {:enable_ipv6, "WASMCLOUD_ENABLE_IPV6", required: false, map: &string_to_bool/1},
+          {:enable_actorfromfs, "WASMCLOUD_ENABLE_ACTORFROMFS",
```

Comment:
What if we renamed to something like `WASMCLOUD_ALLOW_FILE_LOAD` (naming is hard) to indicate that we're just allowing this host to load generically from a file? Would be a good thing for providers too